components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -913,6 +915,35 @@ pub fn postprocess_extended_package_list(_req: &HttpRequest,
             .body(body)
 }
 
+pub fn postprocess_package_settings(_req: &HttpRequest,
+                                    packages: &[BuilderPackageSetting],
+                                    count: i64,
+                                    pagination: &Query<Pagination>)
+                                    -> HttpResponse {
+    let (start, _) = helpers::extract_pagination(pagination);
+    let pkg_count = packages.len() as isize;
+    let stop = match pkg_count {
+        0 => count,
+        _ => (start + pkg_count - 1) as i64,
+    };
+
+    debug!("postprocessing package list, start: {}, stop: {}, total_count: {}",
```

Comment:
nit: we might wish to modify this debug just slightly to be something like `postprocessing package settings list` to distinguish it from this other very similar debug https://github.com/habitat-sh/builder/blob/ba06a2ee9b6e8940fbf67e3610ee7aa9a7e3861f/components/builder-api/src/server/resources/pkgs.rs#L870